### DISCUSSION TITLE
Validating YAML file remotely

### DISCUSSION THREAD

**Discussion by roylisto** [cue-lang/cue#1081] [Q&A]
Hi, does CUE have the ability to validating the YAML files remotely by putting URL?
for example 
`cue vet "https://my-yaml-url-here" file.cue`

**Comment by mpvl** [comment]
This not supported, but you can try curl-ing it and piping it like
```
curl ... | cue vet file.cue yaml: -
```

**Reply by santoshkal** [reply]
Hi, just looking for any updates on this.
Does CUE  now have this ability or we still need to use curl?

Thanks
